### PR TITLE
Set max-nexted-path during mirroring

### PR DESCRIPTION
**Pull request by betoredhat**
This will allow to properly render the ICSP/IDMS when the operators are mirrored inside an organization, like in prega/<operators>

Test-Hint: no-check

### CODE REVIEW

**Comment by betoredhat** [comment]
recheck

File: roles/mirror_from_directory/README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ mfd_local_registry     | Yes      |
 mfd_operators_dir      | Yes      |                                               | Directory containing the tarball files generated by oc-mirror
 mfd_delete_workspace   | No       | True                                          | Deletes the `oc-mirror` workspace generated during the mirroring
 mfd_local_registry_path| No       |                                               | Path/Org in the local registry where the images will be mirrored into
+mfd_max_nested_paths   | No       | 3                                             | Maximum nested paths on target registry
```
[the remark this answers is not in the document]
Comment (betoredhat) [reply roles/mirror_from_directory/README.md:13]: I have no idea about the limit. As you saw the doc is not very clear, the default is 2 and that is not working properly for cases where the mirroring is performed inside an organization with 3 nesting levels.

**Comment by betoredhat** [comment]
Failure is because the registry.redhat.io/openshift4/numaresources-rhel9-operator@sha256:1ec7ed189a0761a940e70d0e89f36619aedf703bdc991db40d31fee0d2eb7c52 is not present in the bundle.  

I found it is present now in GA catalog

File: roles/mirror_from_directory/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (tonyskapunk) [reply roles/mirror_from_directory/README.md:13]: My suggestion would be then to set the default to 2 and set the 3 paths where is needed

**Review by manurodriguez — APPROVED** [review]
LGTM

**Comment by betoredhat** [comment]
recheck

**Pull request merged**